modules/admin/admin-impl/src/main/java/com/enonic/xp/admin/impl/rest/resource/project/json/ProjectGraphJson.java

```diff
@@ -0,0 +1,19 @@
+package com.enonic.xp.admin.impl.rest.resource.project.json;
+
+import java.util.stream.Collectors;
+
+import com.google.common.collect.ImmutableList;
+
+import com.enonic.xp.project.ProjectGraph;
+import com.enonic.xp.support.AbstractImmutableEntityList;
+
+public final class ProjectGraphJson
+    extends AbstractImmutableEntityList<ProjectGraphEntryJson>
```

Comment:
Is it OK to extend lmmutavleList here?

Reply:
we do the same with AccessControlEntriesJson